src/components/calcite-button/calcite-button.stories.js

```diff
@@ -2,9 +2,21 @@ import { storiesOf } from "@storybook/html";
 import { withKnobs, text, boolean, select } from "@storybook/addon-knobs";
 import { images24 } from "@esri/calcite-ui-icons/js/images24";
 import { darkBackground, parseReadme } from "../../../.storybook/helpers";
+import * as icons from "../../../node_modules/@esri/calcite-ui-icons";
 import readme from "./readme.md";
 const notes = parseReadme(readme);
 
+// we can get all unique icon names from all size 16 non-filled icons.
+const iconNames = Object.keys(icons)
```

Comment:
I could create a utility for icons since this is now in two stories (`icon` and here).

Reply:
👍 That'd be useful for upcoming components that accept icons as well.

Reply:
agreed!!

Reply:
Created issue: https://github.com/Esri/calcite-components/issues/336https://github.com/Esri/calcite-components/issues/336